Unreal Launcher/Unreal Launcher/MainWindow.xaml.cs

```diff
@@ -49,19 +49,19 @@ public MainWindow()
 					{
 						project.Init();
 
-						if (project.projectInitialised)
+						if (project.ProjectInitialised)
 						{
 							AddProjectTab(project);
 						}
 						else
 						{
-							InvalidProjects.Add(projectstr);
+							invalidProjects.Add(projectstr);
 						}
 					}
 				}
 
 				// Loop through projects that could not be found on disk and remove them.
-				foreach (string invalidProjectstr in InvalidProjects)
+				foreach (string invalidProjectstr in invalidProjects)
```

Comment:
`invalidProjectstr` would be better simply as `invalidProject` (or if that's already defined elsewhere, then at least use a better descriptor than "str")